src/supermarket/config/initializers/paperclip.rb

```diff
@@ -21,7 +17,8 @@
       },
       path: path,
       bucket: ENV['S3_BUCKET'],
-      s3_protocol: ENV['PROTOCOL']
+      s3_protocol: ENV['PROTOCOL'],
+      s3_region: ENV['S3_REGION']
     }
 
     if ENV['S3_PRIVATE_OBJECTS'].present?
```

Comment:
Surveying the current uses of `.present?`

* [`S3_PATH`](https://github.com/chef/supermarket/blob/66189112c5b95689866df565a6bda2679291dea9/src/supermarket/config/initializers/paperclip.rb#L10) :: 🆗  because if a path is given, we want to use it.
* [`S3_PRIVATE_OBJECTS `](https://github.com/chef/supermarket/blob/66189112c5b95689866df565a6bda2679291dea9/src/supermarket/config/initializers/paperclip.rb#L27) :: ⚠️  because it will cause unexpected behavior if someone sets it to `"false"`.
* [`CDN_URL `](https://github.com/chef/supermarket/blob/66189112c5b95689866df565a6bda2679291dea9/src/supermarket/config/initializers/paperclip.rb#L33) :: 🆗  because if the URL is given, we want to use it.

Boiled down, I'm of the opinion that `.present?` is good for environment variables that have values to use to build paths, authenticate to things and such. But `S3_PRIVATE_OBJECTS` is acting like a feature flag and it doesn't take a value of "false" in a way that would turn that feature off.